doc/classes/CameraServer.xml

```diff
@@ -6,7 +6,7 @@
 	<description>
 		The [CameraServer] keeps track of different cameras accessible in Godot. These are external cameras such as webcams or the cameras on your phone.
 		It is notably used to provide AR modules with a video feed from the camera.
-		[b]Note:[/b] This class is currently only implemented on macOS and iOS. To get a [CameraFeed] on iOS, the camera plugin from [url=https://github.com/godotengine/godot-ios-plugins]godot-ios-plugins[/url] is required. On other platforms, no [CameraFeed]s will be available.
+		[b]Note:[/b] This class is currently only implemented on Linux, macOS, and iOS. To get a [CameraFeed] on iOS, the camera plugin from [url=https://github.com/godotengine/godot-ios-plugins]godot-ios-plugins[/url] is required. On other platforms, no [CameraFeed]s will be available.
```

Comment:
Isn't `CameraFeed` implemented for Linux now?

Reply:
Not sure what you mean. Maybe add suggestion?
Thanks for other comments. :)

Reply:
So what was confusing to me is that it says first:
`To get a [CameraFeed] on iOS, the camera plugin from [url=https://github.com/godotengine/godot-ios-plugins]godot-ios-plugins[/url] is required.`

And then:
`On other platforms, no [CameraFeed]s will be available.`

Maybe the whole sentence should be:
`[b]Note:[/b] This class is currently only implemented on Linux, macOS, and iOS. On other platforms, no [CameraFeed]s will be available.`

And then continue with: 
`To get a [CameraFeed] on iOS, the camera plugin from [url=https://github.com/godotengine/godot-ios-plugins]godot-ios-plugins[/url] is required.`